openMediaAppearIn/readme.md

```diff
@@ -1,5 +1,5 @@
-# openMediaVideoConferencing.html
-Basic demo of open media conversation routing.
+# openMediaAppearIn.html
+Basic demo of open media conversation routing by glueing [appear.in](https://appear.in/)
```

Comment:
Changed it to: ... routing through integration with appear.in

Some references for the spelling of 'glueing': 
[thefreedictionary](http://www.thefreedictionary.com/glueing)
[merriam-webster](https://www.merriam-webster.com/dictionary/glue)